README.md

```diff
@@ -524,6 +525,18 @@ repository.
 OmniSharp is written in C# and has to be compiled. The `install.sh` script takes
 care of this if you pass `--omnisharp-completer` as an argument.
 
+### Go semantic completion
+
+YCM uses [Gocode][] to provide semantic completion for Go. It's used as a git
```

Comment:
Mentioning that YCM pulls in gocode as a submodule is unnecessary; anyone not calling `git
submodule update --init --recursive` (through Vundle or directly) is going to have a completely broken YCM no matter what (ycmd is used as a submodule for instance). The YCM build process checks for this and issues error messages, so it can't silently bite the user.

So you can just delete this whole section. The only updates that are needed are mentioning the `--gocode-completer` flag.

I've now noticed that the Python & C# semantic completion section also talk about submodules; I'll be removing that.